.circleci/config.yml

```diff
@@ -33,6 +33,38 @@ jobs:
           command: |
               flake8 --version
               flake8 habitat/. habitat_baselines/. examples/. tests/. setup.py
+  pre-commit:
+    docker:
+      - image: circleci/python:3.6
+    working_directory: ~/repo/
```

Comment:
Where this `repo` dir is created?

Reply:
@mathfac automatically in the docker config apparently. This circelci is already live for habitat-sim